crates/compiler/builtins/bitcode/src/utils.zig

```diff
@@ -156,7 +158,18 @@ pub fn increfRcPtrC(ptr_to_refcount: *isize, amount: isize) callconv(.C) void {
         // As such, we do not need to cap incrementing.
         switch (RC_TYPE) {
             Refcount.normal => {
+                const old = @intCast(usize, ptr_to_refcount.*);
                 ptr_to_refcount.* += amount;
+                const new = @intCast(usize, ptr_to_refcount.*);
+
+                if (DEBUG_INCDEC and builtin.target.cpu.arch != .wasm32) {
+                    const stdout = std.io.getStdOut().writer();
+
+                    const oldH = old - REFCOUNT_ONE + 1;
+                    const newH = new - REFCOUNT_ONE + 1;
+
+                    stdout.print("| increment {*}: {} + {} = {}!\n", .{ ptr_to_refcount, oldH, amount, newH }) catch unreachable;
```

Comment:
nit: for these type of things, I would use `std.debug.print` instead of instantiating stdout and such. It essentially has builtin setup and catch unreachable.

Reply:
TIL